# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/BrowserBridgeParent.h
# Commit: d77d76d37d4d
# Full Hash: d77d76d37d4dccf39c12fa81c970088b5388d0f1
# Author: Hiroyuki Ikezoe <hikezoe@mozilla.com>
# Date: 2019-05-13 08:22:56
# Regressor Bug: 1518919
# File Overlap Count: 1
# Description:
#   Bug 1518919 - Propagate CSS visibility information to descendant documents via IPC call for fission. r=jwatt
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D30460
# ==============================================================================

diff -r 02b3eb8a1a13 -r d77d76d37d4d dom/ipc/BrowserBridgeParent.h
--- a/dom/ipc/BrowserBridgeParent.h	Thu May 09 07:37:03 2019 +0000
+++ b/dom/ipc/BrowserBridgeParent.h	Mon May 13 01:49:07 2019 +0000
@@ -63,6 +63,9 @@
 
   mozilla::ipc::IPCResult RecvDeactivate();
 
+  mozilla::ipc::IPCResult RecvSetIsUnderHiddenEmbedderElement(
+      const bool& aIsUnderHiddenEmbedderElement);
+
   void ActorDestroy(ActorDestroyReason aWhy) override;
 
  private: